src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```diff
@@ -17,6 +17,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Trimmer defaults that depend on user-definable settings.
         This must be configured before it's initialized in the .NET SDK targets (which are imported by the Razor SDK). -->
     <SuppressTrimAnalysisWarnings Condition="'$(SuppressTrimAnalysisWarnings)' == '' And '$(TrimmerDefaultAction)' != 'link'">true</SuppressTrimAnalysisWarnings>
+
+    <!-- Similarly these feature switches must be configured before they are initialized in imported SDKs -->
+    <JsonSerializerIsReflectionEnabledByDefault Condition="'$(JsonSerializerIsReflectionEnabledByDefault)' == ''">true</JsonSerializerIsReflectionEnabledByDefault>
```

Comment:
AFAIK blazor apps don't run here. So best way to test it is to locally run blazor app with the built SDK.

Reply:
This can't be disabled for blazor apps, if the user disables this, we should emit an error instead in the context of blazor webassembly apps or simply ignore the user choice.

The framework is dependent on this feature for basic functionality (JS interop)